get_operator_versions.sh

```diff
@@ -8,7 +8,8 @@ IMG="registry.redhat.io/redhat/${INDEX}-operator-index:v${OCP_MAJ_VER}"
 echo IMG=${IMG}
```

Comment:
Is the `get_operator_versions.sh` used as part of the build pipeline? Or is it just to enable local testing?